update.php

```diff
@@ -173,3 +175,12 @@ function update_1245() {
 
 	return UPDATE_SUCCESS;
 }
+
+function update_1247() {
+	// Removing hooks with the old name
+	dba::e("DELETE FROM `hook`
+WHERE `hook` LIKE 'plugin_%'");
```

Comment:
the indentation looks wrong in the github diff.

Reply:
It's a habit I formed after having debugged countless SQL query strings (before statements). I wouldn't indent the queries at their current level because the output would be affected by where they would be in the code. Instead, I resolved to deindent them manually, which made them stand out in my code and look nicer when debugging.

Reply:
They are "deindented" automatically by the dba class. So this doesn't affect the query.

Reply:
I know, like I said it's a personal habit, I'll try to be more mindful of it.